src/main/java/ch/heigvd/mcr/bridgehack/MapState.java

```diff
@@ -0,0 +1,166 @@
+package ch.heigvd.mcr.bridgehack;
+
+import ch.heigvd.mcr.bridgehack.player.Player;
+import ch.heigvd.mcr.bridgehack.player.races.Dwarf;
+import ch.heigvd.mcr.bridgehack.player.races.Human;
+import ch.heigvd.mcr.bridgehack.player.roles.Hunter;
+import ch.heigvd.mcr.bridgehack.player.roles.Knight;
+import ch.heigvd.mcr.bridgehack.player.roles.Wizard;
+import org.newdawn.slick.*;
+import org.newdawn.slick.Graphics;
+import org.newdawn.slick.state.BasicGameState;
+import org.newdawn.slick.state.StateBasedGame;
+
+import java.awt.Font;
+import java.util.LinkedList;
+
+public class MapState extends BasicGameState {
+    public static final int ID = 3;
+
+    private LinkedList<Map> maps;
+    private Map map;
+    private Player player;
+    // Temporary
+    private LinkedList<Player> enemies = new LinkedList<>();
+
+    private String notification = "";
+    private boolean attacking;
+    private TrueTypeFont ttf;
+    private boolean turnIsOver = false;
+    private int turn = 0;
+    private int counter = 0;
+
+    @Override
+    public void init(GameContainer gameContainer, StateBasedGame stateBasedGame) throws SlickException {
+        maps = new LinkedList<>();
+        maps.add(new Map(1, false));
+        maps.add(new Map(2, true));
+        map = maps.get(0);
```

Comment:
3rd map is missing

Reply:
But don't add it, since @Tatallias has done it in #31.